01-bash-scripting/01_cpu_and_memory_usage.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+get_user_cpu_usage() {
+  top -l 1 | awk '/CPU usage/ {print $3}' | cut -d '%' -f 1
+}
+
+get_sys_cpu_usage() {   
+  top -l 1 | awk '/CPU usage/ {print $5}' | cut -d '%' -f 1
+}
+
+get_memory_usage() {
+  vm_stat | awk '/Pages active/ {print $3*4096*0.000000001}' # pages to gigabytes
+}
+
+user_cpu_usage=$(get_user_cpu_usage)
+sys_cpu_usage=$(get_sys_cpu_usage)
+memory_usage=$(get_memory_usage)
```

Comment:
You should always quote strings containing variables, command substitutions, spaces or shell meta characters.
These are recommended guidelines, not mandatory regulation. 

You can read more: https://google.github.io/styleguide/shellguide.html#s5.7-quoting